src/snowflake/snowpark/modin/plugin/docstrings/io.py

```diff
@@ -474,6 +474,179 @@ def read_orc():
+
+    date_format : str or dict of column -> format, default None
+        If used in conjunction with parse_dates, will parse dates according to this format. For anything more complex, please read in as object and then apply to_datetime() as-needed.
+
+    thousands : str, default None
+        Thousands separator for parsing string columns to numeric. Note that this parameter is only necessary for columns stored as TEXT in Excel, any numeric columns will automatically be parsed, regardless of display format.
+    decimal : str, default ‘.’
+        Character to recognize as decimal point for parsing string columns to numeric. Note that this parameter is only necessary for columns stored as TEXT in Excel, any numeric columns will automatically be parsed, regardless of display format.(e.g. use ‘,’ for European data).
+
+    comment : str, default None
+        Comments out remainder of line. Pass a character or characters to this argument to indicate comments in the input file. Any data between the comment string and the end of the current line is ignored.
+    skipfooter : int, default 0
+        Rows at the end to skip (0-indexed).
+    storage_options : dict, optional
+        Extra options that make sense for a particular storage connection, e.g. host, port, username, password, etc. For HTTP(S) URLs the key-value pairs are forwarded to urllib.request.Request as header options. For other URLs (e.g. starting with “s3://”, and “gcs://”) the key-value pairs are forwarded to fsspec.open. Please see fsspec and urllib for more details, and for more examples on storage options refer here.
+    dtype_backend : {‘numpy_nullable’, ‘pyarrow’}, default ‘numpy_nullable’
+        Back-end data type applied to the resultant DataFrame (still experimental). Behaviour is as follows:
+        - "numpy_nullable": returns nullable-dtype-backed DataFrame (default).
+        - "pyarrow": returns pyarrow-backed nullable ArrowDtype DataFrame.
+
+    engine_kwargs : dict, optional
+        Arbitrary keyword arguments passed to excel engine.
+
+    Returns
+    -------
+    DataFrame or dict of DataFrames
+        DataFrame from the passed in Excel file. See notes in sheet_name argument for more information on when a dict of DataFrames is returned.
+
+    See also
+    --------
+    DataFrame.to_excel
+        Write DataFrame to an Excel file.
+    DataFrame.to_csv
+        Write DataFrame to a comma-separated values (csv) file.
+    read_csv
+        Read a comma-separated values (csv) file into DataFrame.
+    read_fwf
+        Read a table of fixed-width formatted lines into DataFrame.
+
+    Notes
```

Comment:
is the engine argument right here? do we use an engine?

Reply:
I believe this is the engine used to actually read/parse the excel file. We do support the parameter - see: https://github.com/snowflakedb/snowpark-python/blob/cf6f3fc5ee0580905f0143fdf7fefc563843d2a4/tests/integ/modin/io/test_read_excel.py#L38